packages/api-demo/src/app.json

```diff
@@ -4,7 +4,8 @@
         "name": "SSF Desktop Wrapper",
         "url": "http://localhost:8080/index.html",
         "uuid": "ssf-desktop-api-openfin-demo",
-        "autoShow": true
+        "autoShow": true,
+        "preload": "http://localhost:8080/scripts/containerjs-api-bundle.js"
```

Comment:
On another note, is there any reason why the api-bundle is being used as the OpenFin preload script? I'd have thought you'd just want to preload the OpenFin specific code?

(and if OpenFin and Electron both use preload scripts, does that mean the api-bundle is now redundant?)